docs/site/Loopback-application-layout-reference.md

```diff
@@ -0,0 +1,123 @@
+---
+lang: en
+title: 'LoopBack 4 Application Layout Reference'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Loopback-application-layout-reference.html
+---
+
+LoopBack 4 application files and directories are in the _application root
+directory_. Within this directory the standard LB4 application structure has
+these sub-directories:
+
+- `src` - TypeScript source code and configuration files.
+- `public` - Client side assets (JavaScript, HTML, and CSS files) for the home
+  page.
+
+{% include tip.html content="By LoopBack naming conventions, artifacts such as models are grouped under their sub-directories. Also, with LoopBack artifact generators, they create class names in CamelCase and file names are in snake-case. For example, if you create a model named `MyUserModel` with the model generator, it creates files `my-user.model.ts` under `src/models`. See [Naming Convention](Command-line-interface.md#naming-convention) for more information.
```

Comment:
Suggestion:
- `with LoopBack artifact generators, they create class names in CamelCase and file names are in snake-case.` -> `LoopBack artifact generators create class names in CamelCase and file names are in snake-case.`


```suggestion
{% include tip.html content="By LoopBack naming conventions, artifacts such as models are grouped under their sub-directories. Also, with LoopBack artifact generators, they create class names in CamelCase and file names are in snake-case. For example, if you create a model named `MyUserModel` with the model generator, it creates the file `my-user.model.ts` under `src/models` directory. See [Naming Convention](Command-line-interface.md#naming-convention) for more information.
```